### ISSUE TITLE
Ninh

### ISSUE THREAD

**Issue by lemanhninh003** [haoict/tiktok-god#33]
Bác ơi tik tok gold vs facebooknoads của bÁc k hoạt động ở chimera/sileo bác ạ mong bác fix

**Comment by steffanjensen** [comment]
i agree we need a fix for sileo package manager.

**Comment by steffanjensen** [comment]
it's really not my domain (litterly) but i will take a look.

**Comment by steffanjensen** [comment]
yeah I just downloaded the .deb. @haoict can you open for upload so we can upload latest release for non cydia users?
